### ISSUE TITLE
Cannot create radio buttons without text or label anymore

### ISSUE THREAD

**Issue by Brow71189** [nion-software/nionui#43]
https://github.com/nion-software/nionui/blob/d4b1db7751da0a747b00f03c0be428d13510152b/nion/ui/Declarative.py#L1449

Since introducing icon radio buttons you cannot create plain radio buttons without text or icon anymore. Attempting this triggers the assertion error linked above.

I think the correct and backwards-compatible behavior would be to first check if an icon if given. If there is, use it, if not, create a "regular" radio button independent from "text" being given or not.

**Comment by cmeyer** [comment]
Can you try this? This ensures one or the other is passed.

```
diff --git a/nion/ui/Declarative.py b/nion/ui/Declarative.py
index 0f7625f..dcacdd3 100755
--- a/nion/ui/Declarative.py
+++ b/nion/ui/Declarative.py
@@ -1436,12 +1436,12 @@ def construct_slider(ui: UserInterface.UserInterface, d: typing.Mapping, handler
 def construct_radio_button(ui: UserInterface.UserInterface, d: typing.Mapping, handler, finishes: typing.Optional[typing.Sequence[typing.Callable[[], None]]]):
     properties = construct_sizing_properties(d)
     widget: typing.Optional[UserInterface.Widget] = None
-    if d.get("text", None):
+    if d.get("text", None) and not d.get("icon", None):
         widget = ui.create_radio_button_widget(properties=properties)
         widget.value = d.get("value", None)
         if handler and d.get("text", None):
             connect_string_value(widget, d, handler, "text", finishes)
-    if d.get("icon", None):
+    if d.get("icon", None) and not d.get("text", None):
         widget = Widgets.IconRadioButtonWidget(ui, properties=properties)
         widget.value = d.get("value", None)
         if handler and d.get("icon", None):
```

**Comment by cmeyer** [comment]
Fixed 0b64d199fca9f4ba381ffaae6360c3056e1e02ec
